mobly/controllers/android_device_lib/snippet_client.py

```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 """JSON RPC interface to Mobly Snippet Lib."""
 
+from __future__ import print_function
```

Comment:
`print_function` is not used?
is this a magical line?

Reply:
it's for python2 compatibility?
print vs print()